src/main/java/org/folio/list/services/UserFriendlyQueryService.java

```diff
@@ -187,41 +190,30 @@ private Object getConditionValue(FieldCondition<?> fieldCondition, EntityType en
       .findFirst()
       .map(Field::getDataType)
       .orElseThrow();
+
     if (!(dataType instanceof DateType)) {
       return fieldCondition.value();
     }
-    LocalDateTime dateTime;
-    String timezone;
-    String dateString = (String) fieldCondition.value();
-    if (dateString.matches(DATE_REGEX)) {
-      dateTime = LocalDate.parse(dateString, DATE_FORMATTER).atStartOfDay();
-    } else if (dateString.matches(DATE_TIME_REGEX)) {
-      dateTime = LocalDateTime.parse(dateString, DATE_TIME_FORMATTER);
+
+    ZoneId tenantTimezone = configurationClient.getTenantTimezone();
+
+    return translationService.formatString(
+      tenantTimezone,
+      "{value, date, short}", "value",
+      handleDateValue((String) fieldCondition.value(), tenantTimezone)
+    );
```

Comment:
The translation service will handle the magic of turning our time (here as an `Instant`) into something nicer for the user, be it `1/15/24` or `15.1.2024` or `২০২৪-০১-১৫` or whatever the locale specifies